test/hotjar.browser.js

```diff
@@ -104,3 +110,27 @@ test('test disclousure is suppressing name', async t => {
     server.close(err => (err ? reject(err) : resolve())),
   )
 })
+
+test('checking suppresion on dashboard once a submission has finished', async t => {
+  const navigationHelper = new NavigationHelper(t)
+  await navigationHelper.fullSubmission(manuscript)
+  await navigationHelper.wait(1000)
+  await t
+    .expect(dashboard.titles, {
+      'data-hj-suppress': '',
+    })
+    .ok()
+})
+
+test('checking suppresion on the thankyou page', async t => {
+  const navigationHelper = new NavigationHelper(t)
+  await navigationHelper.paricialSubmissionThankyou(manuscript)
+  await navigationHelper.consentDisclosure()
+  await navigationHelper.submit()
+  await navigationHelper.accept()
+  await t
```

Comment:
This is quite a lot of process to check that a data attribute exists on an element on the `ThankYou` page. Could this check not be achieved through a unit test instead?

Reply:
And maybe a single browser test for checking suppression? (multiple assertions as it progresses)

Reply:
@jyenes I've added a unit test for this to keep the coverage up.

Reply:
thanks @Cooryd